compiler/src/iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenOps.td

```diff
@@ -101,7 +102,17 @@ def IREECodegen_ExtractStridedMetadataOp : Op<IREECodegen_Dialect, "extract_stri
   }];
 }
 
+def IREECodegen_NullPointerOp :
+     Op<IREECodegen_Dialect, "null_pointer", [Pure]> {
+  let summary = "Returns a null_pointer value.";
 
+  let description = [{
+    Returns a null_pointer value.
+  }];
```

Comment:
Can we drop this if the description doesn't add anything new beyond the summary?